packages/allure-playwright/src/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
why is this needed? There are `eslint-disable-line` below

Reply:
Oops, my bad 🤪